src/components/form/EncounterForm.jsx

```diff
@@ -155,27 +151,21 @@ const mapStateToProps = (state, props) => {
         formInstanceId: props.formInstanceId,
         patient: props.patient,
         encounter: props.encounter,
-        encounterRole: props.encounterRole ? props.encounterRole : null,
+        encounterRole: props.encounterRole || props.encounterRole || null,
```

Comment:
The `|| null`s probably shouldn't be necessary, but who knows, maybe something somewhere downstream differentiates between `""` and `null`

Reply:
why the double props.encounterRole?